### PR TITLE
Fix UI

### LINKED ISSUE #120: [PE-D][Tester E] View command UI layout
After entering the view command in some modules and having both semester boxes expanded, the 2nd semester box can not be fully expanded out.

Maybe the team can make it such that only one tab can be expanded at one time?

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: jialatteo/ped#2

### PR DESCRIPTION
**Pull request by SeekSaveServe**
- [x] Limit the main window to at least 75% of the screen size

- [x] Set focus o command text field on TAB

- [x] Fix the auto 'shrinking' of accordions when a second accordion is being expanded 

https://user-images.githubusercontent.com/53963433/199150488-a57cb0d8-971e-4708-8ee1-fde52aca64f1.mp4

Not sure if disabling focus on the module panels is the best way to do this. What do yall think?

Closes #120 #122 #135 #124

### CODE REVIEW

**Comment by zupey** [comment]
I tried out the new changes made in this PR.  Here are some comments for discussion.

1. Is setting the main window to 75% of screen size the best solution? Now cannot split screen cos it takes up most of the screen space.
2. I cant seem to use tab to focus on the command line.  To replicate, I clicked on one of the pills in module list (e.g. 4MCs). Or even worse, at this page: (`view cs1101s` and open one of the accordions)
image
3. There seems to be some unexpected behaviour as follows:


https://user-images.githubusercontent.com/53803800/199178459-ef75babb-a16d-456e-8b16-f43c47902a8e.mp4

File: src/main/java/nus/climods/ui/module/components/SemesterPill.java [hunk 1]
```diff
@@ -43,5 +44,9 @@ public SemesterPill(SemestersEnum semester, String bgColor, String textColor) {
      */
     public SemesterPill(SemestersEnum semester, String bgColor, String textColor, int fontSize) {
         super(semester.toString(), bgColor, textColor, fontSize);
+        setUI();
+    }
+    private void setUI() {
+        super.setFocusTraversable(false);
```
Comment (shenyih0ng) [inline src/main/java/nus/climods/ui/module/components/SemesterPill.java:50]: Maybe we can just make all `Pill` non focus traversable?

File: src/main/java/nus/climods/ui/common/CommandBox.java [hunk 2]
```diff
@@ -30,8 +31,13 @@ public CommandBox(CommandSession commandSession) {
         super(FXML);
         this.commandSession = commandSession;
         // calls #setStyleToDefault() whenever there is a change to the text of the command box.
+        commandTextField.requestFocus();
```
Comment (shenyih0ng) [inline src/main/java/nus/climods/ui/common/CommandBox.java:34]: I don't this line is necessary?

File: src/main/java/nus/climods/ui/common/CommandBox.java [hunk 3]
```diff
@@ -30,8 +31,13 @@ public CommandBox(CommandSession commandSession) {
...
         commandTextField.textProperty().addListener((unused1, unused2, unused3) -> setStyleToDefault());
         commandTextField.setOnKeyPressed(event -> {
+            if (event.getCode().equals(KeyCode.TAB)) {
+                commandTextField.requestFocus();
+            }
```
Comment (shenyih0ng) [inline src/main/java/nus/climods/ui/common/CommandBox.java:39]: The issue raised by @zupey could be related to https://stackoverflow.com/a/32807849

TLDR, events will not be passed to UI nodes unless they are focused

**Review by sarrrdin — APPROVED** [review]
Note that you are merging directly from your master branch... overall UI works well on my device, LGTM 👍
